.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/sifive/example-hello.git
 [submodule "freedom-metal"]
 	path = freedom-metal
-	url = https://github.com/sifive/freedom-metal.git
+	url = git@github.com:sifive/freedom-metal.git
```

Comment:
This change should be backed out, we want HTTPS URLs in .gitmodules.

Reply:
Will be reverted.